Assets/Labrys/FeatureEditor/FeatureAsset.cs

```diff
@@ -0,0 +1,372 @@
+﻿using Labrys.Generation;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Labrys.FeatureEditor
+{
+	/// <summary>
+	/// A file-backed representation of a Feature for use with the Feature Editor
+	/// </summary>
+	[CreateAssetMenu(menuName = "Labrys/Feature", fileName = "New Feature")]
+	public class FeatureAsset : ScriptableObject, ISerializationCallbackReceiver
+	{
+		[Serializable]
+		public class Section
```

Comment:
Section as a non-static class provides extra readability, imo.  Makes it easier to see that a section is being manipulated, rather than just a bare-bones string with little context. Plus, if we ever plan to expand the meta-data included in sections, then with the current design, additions would be easy.

Propose no change.

Reply:
Works for me